src/App.tsx

```diff
@@ -1,19 +1,23 @@
 import { Suspense, lazy, useEffect, useState } from "react";
 import { BrowserRouter, Routes, Route, Navigate } from "react-router-dom";
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
-import { ReactQueryDevtools } from "@tanstack/react-query-devtools";
 import { CookiesProvider, useCookies } from "react-cookie";
 import { CookieSetOptions } from "universal-cookie";
 import { v4 as uuidv4 } from "uuid";
+import { WrenchScrewdriverIcon } from "@heroicons/react/24/outline";
+import { Resizable } from "re-resizable";
 
 import "@/index.css";
 
 import { ChooseSession, Edit, NoMatch } from "@/components";
+import { DevOptions } from "@/components/Edit";
 
-const ReactQueryDevtoolsProduction = lazy(() =>
+// This ensures that we don't unnecessarily load the tools in production.
+// https://tanstack.com/query/v4/docs/react/devtools#devtools-in-production
+const ReactQueryDevtoolsPanel = lazy(() =>
```

Comment:
Will it still be possible to load the tools in production? It would be nice to have that option if needed.

Reply:
Never mind, I was confused that this was a diff and though that something had changed.